satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -452,6 +483,19 @@ def FakeFCIFileHandlerHRFI_fixture():
         }
         yield param_dict
 
+@pytest.fixture()
+def FakeFCIFileHandlerAF_fixture(channel,resolution):
+    """Get a fixture for the fake AF filehandler, including channel and file names."""
```

Comment:
Please expand here specifying that the AF fixture needs channel and resolution since one file contains only one channel in this case.